src/components/elements/NodeCard.tsx

```diff
@@ -1,6 +1,8 @@
 import { useRouter } from "next/router";
-import { Card, Row, Col, Typography } from "antd";
+import Image from "next/image";
+import { Card, Row, Col, Tooltip, Typography } from "antd";
```

Comment:
Agreed. The point of these premade components and Next specific implementations is to take some of the work of "reinventing the wheel" off of us - these teams have given us battle tested code (plus if we have issues with the code, others will too, meaning it will probably be resolved sooner than if we hand rolled our own)